### ISSUE TITLE
Cant get it to work

### ISSUE THREAD

**Issue by johro897** [Sian-Lee-SA/honeycomb-menu#7]
Tried your examples and made multiple tries without any success. Just cant get it to work.
As far as I can tell, no error message.

Used chrome

Tried with installing with HACS, which only installs the honeycomb-menu.js and also tried manually and installed all source files.
No matter what, not working.

**Comment by dotKrad** [comment]
Did you make it work ?

**Comment by Sian-Lee-SA** [comment]
Are you able to open the inspector in chrome and check the network tab? in that tab after refreshing the page and record toggled (red) then you should be able to see if the honeycomb.js file is being loaded. Please advise if it does or doesn't load. thanks

**Comment by dotKrad** [comment]
Actually it's been loaded two times, 1 says Status Code: 200 OK (from ServiceWorker) and the other one Status Code: 200 OK

**Comment by johro897** [comment]
I actually removed it after many tries, will try once more.
Just need to branch out first

**Comment by johro897** [comment]
> Are you able to open the inspector in chrome and check the network tab? in that tab after refreshing the page with the record button toggled (red) then you should be able to see if the honeycomb.js file is being loaded. Please advise if it does or doesn't load. thanks

Yes it is loaded, but still nothing happens.
Either instructions on how to use is wrong or something in code.
I just made a simple button according to the info:

`type: 'custom:button-card'
entity: light.lampan_pa_kontoret
tap_action:
  action: toggle
show_name: false
honeycomb:
  buttons:
    - icon: 'mdi:lightbulb'
      tap_action:
        action: more-info
    - icon: 'mdi:information-variant'
      tap_action:
        action: more-info`

**Comment by Sian-Lee-SA** [comment]
> Actually it's been loaded two times, 1 says Status Code: 200 OK (from ServiceWorker) and the other one Status Code: 200 OK

It should only load once, for some reason it is dual loading. Did you happen to load through HACS and manual? I believe there maybe a clash but first to be sure, we need to remove one of the loadings. Check settings -> Lovelace Dashboard -> Resources and if you find honeycomb-menu there, then remove it for now then reload and see if it still loads atleast once. Hopefully it should work once the dual loading is out of the equation.

**Comment by Sian-Lee-SA** [comment]
> > Are you able to open the inspector in chrome and check the network tab? in that tab after refreshing the page with the record button toggled (red) then you should be able to see if the honeycomb.js file is being loaded. Please advise if it does or doesn't load. thanks
> 
> Yes it is loaded, but still nothing happens.
> Either instructions on how to use is wrong or something in code.
> I just made a simple button according to the info:
> 
> ```
> type: 'custom:button-card'
> entity: light.lampan_pa_kontoret
> tap_action:
>   action: toggle
> show_name: false
> honeycomb:
>   buttons:
>     - icon: 'mdi:lightbulb'
>       tap_action:
>         action: more-info
>     - icon: 'mdi:information-variant'
>       tap_action:
>         action: more-info
> ```

I assume the button itself is working and showing up fine?

**Comment by rebmemer** [comment]
Not to hijack this but it seems to have gone a bit quiet so I'll chime in :)

Firstly, the honeycomb looks great - exactly what I wanted.  Only issue is I also can't get it to work?

I've installed honeycomb via HACS and already had the card-modder resource in place, I did a few tests with my own honeycomb definition but couldn't get anything to appear so I took your example above.  My situation is probably a little more vanilla because after checking (in the Chrome dev tools) I can see honeycomb.js has only loaded once, without errors.

Using your sample code above, the button appears correctly showing the 'light' mdi, the toggle (tap) action works by turning on the associated entity (in my case light.desk_lamp) but clicking/holding/anything doesn't show any sort of honeycomb or provide any change in behaviour apart from changing the entity state (on/off).

Happy to test anything you want, I'm eager to get this working :)

Thanks
